lib/survey_gizmo/configuration.rb

```diff
@@ -45,16 +56,25 @@ def initialize
       @api_token = ENV['SURVEYGIZMO_API_TOKEN'] || nil
       @api_token_secret = ENV['SURVEYGIZMO_API_TOKEN_SECRET'] || nil
 
-      @api_url = DEFAULT_REST_API_URL
       @api_version = DEFAULT_API_VERSION
       @results_per_page = DEFAULT_RESULTS_PER_PAGE
 
       @timeout_seconds = DEFAULT_TIMEOUT_SECONDS
       @retry_attempts = DEFAULT_RETRIES
       @retry_interval = DEFAULT_RETRY_INTERVAL
+      self.region = DEFAULT_REGION
 
       @logger = SurveyGizmo::Logger.new(STDOUT)
       @api_debug = ENV['GIZMO_DEBUG'].to_s =~ /^(true|t|yes|y|1)$/i
     end
+
+    def region=(region)
+      region_infos = SURVEY_GIZMO_APIS[region]
+      fail 'Unknown server!' unless region_infos
```

Comment:
You'd better put `ArgumentError.new("Unknown region: #{region}")` in the exception, because it's a wrong argument and now we know why